forloop_modules/function_handlers/database_handlers.py

```diff
@@ -10,21 +10,46 @@
 
 import forloop_modules.flog as flog
 import forloop_modules.queries.node_context_requests_backend as ncrb
+import forloop_modules.globals.db_connection as dbc
 
 from forloop_modules.function_handlers.auxilliary.node_type_categories_manager import ntcm
 from forloop_modules.function_handlers.auxilliary.form_dict_list import FormDictList
 from forloop_modules.globals.variable_handler import variable_handler
 from forloop_modules.globals.database_utilities_handler import duh
+from forloop_modules.globals.active_entity_tracker import aet
 from forloop_modules.globals.docs_categories import DocsCategories
 from forloop_modules.function_handlers.auxilliary.docs import Docs
 from forloop_modules.function_handlers.auxilliary.abstract_function_handler import AbstractFunctionHandler
 from forloop_modules.function_handlers.auxilliary.data_types_validation import validate_input_data_types
+from forloop_modules.utils.encryption import decrypt_text, convert_base64_private_key_to_rsa_private_key
+from forloop_modules.redis.redis_connection import kv_redis, create_redis_key_for_project_db_private_key
 
 def parse_comboentry_input(input_value: list[str]):
     input_value = input_value[0] if isinstance(input_value, list) and len(input_value) > 0 else input_value
     
     return input_value
 
+def get_all_databases_in_project():
+    response = ncrb.get_all_databases()
```

Comment:
change to "get_all_databases_by_project_uid" -> put it in NCRB and remove this function from database handlers